assets/src/components/v2/elevator/closures.tsx

```diff
@@ -6,15 +6,20 @@ import makePersistent, {
   WrappedComponentProps,
 } from "Components/v2/persistent_wrapper";
 import PagingIndicators from "Components/v2/elevator/paging_indicators";
-import {
-  type StationWithClosures,
-  type Closure,
-} from "Components/v2/elevator/types";
+import { type StationWithClosures } from "Components/v2/elevator/types";
 import useIntervalPaging from "Hooks/v2/use_interval_paging";
-import NormalService from "Images/svgr_bundled/normal-service.svg";
+import CalendarIcon from "Images/svgr_bundled/calendar.svg";
+import CalendarAlertIcon from "Images/svgr_bundled/calendar-alert.svg";
+import NormalServiceIcon from "Images/svgr_bundled/normal-service.svg";
 import AccessibilityAlert from "Images/svgr_bundled/accessibility-alert.svg";
+import { hasOverflowX } from "Util/utils";
 import { CLOSURES_PAGING_INTERVAL_MS } from "./constants";
 
+type UpcomingClosureInfo = {
+  banner: { title: string; trailer: string | null };
```

Comment:
Nit, in part b/c I see how many lines it would require changing -- I found the variable `trailer` confusing at first b/c I haven't seen that used before to refer to trailing words as it's doing here. I'd prefer something like `suffix` or `postfix`, and then when it's used elsewhere it could be `banner_suffix`

Reply:
Hmm, I like `postfix`, hadn't considered that. I didn't want to call it `suffix` (since that sort of implies something tacked onto the end of running text, which is not the case in the details view) nor `footer` (since that implies a separate block element, which is not the case in the banner). I don't mind changing it since I _also_ didn't love `trailer`.